### DISCUSSION TITLE
Testing configuration options that affect CLI output

### DISCUSSION THREAD

**Discussion by giraugh** [dotansimha/graphql-code-generator#9114] [General]
Started investigating a fix for #8973 and noticed that the `listr` output in `packages/graphql-codegen-cli/src/codegen.ts` is silenced during tests. As a result, testing whether output is silenced when `errorsOnly` is set becomes tricky. I can continue working on this without a test or investigate alternative ways of silencing logging output during other tests.

Any input is appreciated, still new to this project :)

**Comment by saihaj** [comment]
I am even happy to find alternatives to `listr` if you have any recommendations?
